static/css/main.css

```diff
@@ -3635,29 +3635,36 @@ header-image {
 
 header-image img {
 	display: block;
-	max-width: 100%; 
-	max-height: 500px; 
+	max-width: 100%;
+	max-height: 500px;
 	margin: 2em auto 2em auto;
 }
 
-vice::before, vimeo::before, youtube::before {
+vice::before, vimeo::before, youtube::before, facebook::before {
 	top: 0;
 	right: 0;
 	left: 0;
 	bottom: 0;
 }
 
-vice, vimeo, youtube {
+vice, vimeo, youtube,facebook {
 	display: block;
 	margin: 30px auto 30px auto;
 	height: 500px;
 }
 
-vice iframe, vimeo iframe, youtube iframe {
+vice iframe, vimeo iframe, youtube iframe{
 	width: 100%;
 	height: 100%;
 }
 
+facebook iframe{
+	display: block;
+	max-width:100%;
+	margin-left: auto;
+  margin-right: auto;
```

Comment:
also keep an eye on formatting when before you do your PR.

best to remove any 

- unnecessary white lines
- ensure formatting is consistent (e.g. margin-right is tabbed left)
- accidental change in convention

I have my css as 
element {
}
deliberately with the space after the element. Either remove all the spaces from the main.css or don't remove them at all.